src/storyMap/components/StoryMapsCard.js

```diff
@@ -51,24 +49,19 @@ const List = withProps(BaseList, {
 
 const GridListItem = withProps(Grid, { component: 'li' });
 
-const StoryMapsHomeCard = props => {
-  const dispatch = useDispatch();
+const StoryMapsCard = ({ onDeleteSuccess, storyMaps, hasAction = true }) => {
```

Comment:
Suggestion: `hasAction` makes sense to `HomeCard` but I don't think it is clear what it does from the `StoryMapsCard` usage. I think what we need here is a `showCreate` prop for `StoryMapsCard` and then you can add something like this before the return:
```js
        const action = useMemo(() =>
          showCreate
            ? {
                label: t('storyMap.home_create'),
                to: 'tools/story-maps/new',
              }
            : null,
          [t, showCreate]
        );
```
And then just pass that action to the `action` prop for `HomeCard`